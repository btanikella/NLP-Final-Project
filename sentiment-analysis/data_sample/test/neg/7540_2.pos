DT NN VBZ VBN IN NNP IN NNP IN RB JJ NNP  DT NN WRB PRP VBD VBN CC VBD RP
DT NN VBZ RB RB JJ : DT NN PRP VBZ DT NN  DT NNS  CC IN PDT DT NNS IN DT NN
IN DT NN PRP VBP DT JJ  JJ  JJ  NN VBP PRP MD VB IN DT NN -LRB- JJ -RRB-  DT NNS -LRB- RB JJ -RRB-  DT NN -LRB- JJ CC JJ -RRB-  CC TO PRP DT VBZ DT JJ VBN TO DT NN POS NN TO DT NN CC PRP$ NNP DT NN IN PRP$ NN TO DT NN CC NN  NNP NNP  TO VB DT NN JJ
