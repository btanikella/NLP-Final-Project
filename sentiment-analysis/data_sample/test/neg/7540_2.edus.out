Sentence #1 (23 tokens):
The film is set in Newcastle on Tyne in north east England, the town where I was born and grew up.
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=8 CharacterOffsetEnd=12 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=13 CharacterOffsetEnd=15 PartOfSpeech=VBZ] [Text=set CharacterOffsetBegin=16 CharacterOffsetEnd=19 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=20 CharacterOffsetEnd=22 PartOfSpeech=IN] [Text=Newcastle CharacterOffsetBegin=23 CharacterOffsetEnd=32 PartOfSpeech=NNP] [Text=on CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=IN] [Text=Tyne CharacterOffsetBegin=36 CharacterOffsetEnd=40 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=41 CharacterOffsetEnd=43 PartOfSpeech=IN] [Text=north CharacterOffsetBegin=44 CharacterOffsetEnd=49 PartOfSpeech=RB] [Text=east CharacterOffsetBegin=50 CharacterOffsetEnd=54 PartOfSpeech=JJ] [Text=England CharacterOffsetBegin=55 CharacterOffsetEnd=62 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=62 CharacterOffsetEnd=63 PartOfSpeech=,] [Text=the CharacterOffsetBegin=64 CharacterOffsetEnd=67 PartOfSpeech=DT] [Text=town CharacterOffsetBegin=68 CharacterOffsetEnd=72 PartOfSpeech=NN] [Text=where CharacterOffsetBegin=73 CharacterOffsetEnd=78 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=79 CharacterOffsetEnd=80 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=81 CharacterOffsetEnd=84 PartOfSpeech=VBD] [Text=born CharacterOffsetBegin=85 CharacterOffsetEnd=89 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=90 CharacterOffsetEnd=93 PartOfSpeech=CC] [Text=grew CharacterOffsetBegin=94 CharacterOffsetEnd=98 PartOfSpeech=VBD] [Text=up CharacterOffsetBegin=99 CharacterOffsetEnd=101 PartOfSpeech=RP] [Text=. CharacterOffsetBegin=101 CharacterOffsetEnd=102 PartOfSpeech=.] 
root(ROOT-0, set-4)
det(film-2, The-1)
nsubjpass(set-4, film-2)
auxpass(set-4, is-3)
prep(set-4, in-5)
pobj(in-5, Newcastle-6)
prep(set-4, on-7)
pobj(on-7, Tyne-8)
prep(set-4, in-9)
advmod(England-12, north-10)
amod(England-12, east-11)
pobj(in-9, England-12)
punct(set-4, ,-13)
det(town-15, the-14)
dobj(set-4, town-15)
advmod(born-19, where-16)
nsubjpass(born-19, I-17)
auxpass(born-19, was-18)
rcmod(town-15, born-19)
cc(born-19, and-20)
conj(born-19, grew-21)
prt(grew-21, up-22)
punct(set-4, .-23)

Sentence #2 (26 tokens):
The film is also fundamentally dishonest - the way it presents the town, the kids, but above all the men of the town.
[Text=The CharacterOffsetBegin=107 CharacterOffsetEnd=110 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=111 CharacterOffsetEnd=115 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=116 CharacterOffsetEnd=118 PartOfSpeech=VBZ] [Text=also CharacterOffsetBegin=119 CharacterOffsetEnd=123 PartOfSpeech=RB] [Text=fundamentally CharacterOffsetBegin=124 CharacterOffsetEnd=137 PartOfSpeech=RB] [Text=dishonest CharacterOffsetBegin=138 CharacterOffsetEnd=147 PartOfSpeech=JJ] [Text=- CharacterOffsetBegin=148 CharacterOffsetEnd=149 PartOfSpeech=:] [Text=the CharacterOffsetBegin=150 CharacterOffsetEnd=153 PartOfSpeech=DT] [Text=way CharacterOffsetBegin=154 CharacterOffsetEnd=157 PartOfSpeech=NN] [Text=it CharacterOffsetBegin=158 CharacterOffsetEnd=160 PartOfSpeech=PRP] [Text=presents CharacterOffsetBegin=161 CharacterOffsetEnd=169 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=170 CharacterOffsetEnd=173 PartOfSpeech=DT] [Text=town CharacterOffsetBegin=174 CharacterOffsetEnd=178 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=178 CharacterOffsetEnd=179 PartOfSpeech=,] [Text=the CharacterOffsetBegin=180 CharacterOffsetEnd=183 PartOfSpeech=DT] [Text=kids CharacterOffsetBegin=184 CharacterOffsetEnd=188 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=188 CharacterOffsetEnd=189 PartOfSpeech=,] [Text=but CharacterOffsetBegin=190 CharacterOffsetEnd=193 PartOfSpeech=CC] [Text=above CharacterOffsetBegin=194 CharacterOffsetEnd=199 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=200 CharacterOffsetEnd=203 PartOfSpeech=PDT] [Text=the CharacterOffsetBegin=204 CharacterOffsetEnd=207 PartOfSpeech=DT] [Text=men CharacterOffsetBegin=208 CharacterOffsetEnd=211 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=212 CharacterOffsetEnd=214 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=215 CharacterOffsetEnd=218 PartOfSpeech=DT] [Text=town CharacterOffsetBegin=219 CharacterOffsetEnd=223 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=223 CharacterOffsetEnd=224 PartOfSpeech=.] 
root(ROOT-0, dishonest-6)
det(film-2, The-1)
nsubj(dishonest-6, film-2)
cop(dishonest-6, is-3)
advmod(dishonest-6, also-4)
advmod(dishonest-6, fundamentally-5)
punct(dishonest-6, --7)
det(way-9, the-8)
dep(dishonest-6, way-9)
nsubj(presents-11, it-10)
rcmod(way-9, presents-11)
det(town-13, the-12)
dobj(presents-11, town-13)
punct(way-9, ,-14)
det(kids-16, the-15)
appos(way-9, kids-16)
punct(way-9, ,-17)
cc(way-9, but-18)
conj(way-9, above-19)
predet(men-22, all-20)
det(men-22, the-21)
pobj(above-19, men-22)
prep(men-22, of-23)
det(town-25, the-24)
pobj(of-23, town-25)
punct(dishonest-6, .-26)

Sentence #3 (78 tokens):
In this film they are all stupid, violent, thieving, thugs.I suppose I could comment on the plot (predictable), the performances (competently unattractive), the direction (lazy and unimaginative), but to me that is all irrelevant compared to the director's insult to the town and its people.Thus the invitation in my summary to the director and writer, Mark Herman, to leave the north east.
[Text=In CharacterOffsetBegin=229 CharacterOffsetEnd=231 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=232 CharacterOffsetEnd=236 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=237 CharacterOffsetEnd=241 PartOfSpeech=NN] [Text=they CharacterOffsetBegin=242 CharacterOffsetEnd=246 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=247 CharacterOffsetEnd=250 PartOfSpeech=VBP] [Text=all CharacterOffsetBegin=251 CharacterOffsetEnd=254 PartOfSpeech=DT] [Text=stupid CharacterOffsetBegin=255 CharacterOffsetEnd=261 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=261 CharacterOffsetEnd=262 PartOfSpeech=,] [Text=violent CharacterOffsetBegin=263 CharacterOffsetEnd=270 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=270 CharacterOffsetEnd=271 PartOfSpeech=,] [Text=thieving CharacterOffsetBegin=272 CharacterOffsetEnd=280 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=280 CharacterOffsetEnd=281 PartOfSpeech=,] [Text=thugs.I CharacterOffsetBegin=282 CharacterOffsetEnd=289 PartOfSpeech=NN] [Text=suppose CharacterOffsetBegin=290 CharacterOffsetEnd=297 PartOfSpeech=VBP] [Text=I CharacterOffsetBegin=298 CharacterOffsetEnd=299 PartOfSpeech=PRP] [Text=could CharacterOffsetBegin=300 CharacterOffsetEnd=305 PartOfSpeech=MD] [Text=comment CharacterOffsetBegin=306 CharacterOffsetEnd=313 PartOfSpeech=VB] [Text=on CharacterOffsetBegin=314 CharacterOffsetEnd=316 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=317 CharacterOffsetEnd=320 PartOfSpeech=DT] [Text=plot CharacterOffsetBegin=321 CharacterOffsetEnd=325 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=326 CharacterOffsetEnd=327 PartOfSpeech=-LRB-] [Text=predictable CharacterOffsetBegin=327 CharacterOffsetEnd=338 PartOfSpeech=JJ] [Text=-RRB- CharacterOffsetBegin=338 CharacterOffsetEnd=339 PartOfSpeech=-RRB-] [Text=, CharacterOffsetBegin=339 CharacterOffsetEnd=340 PartOfSpeech=,] [Text=the CharacterOffsetBegin=341 CharacterOffsetEnd=344 PartOfSpeech=DT] [Text=performances CharacterOffsetBegin=345 CharacterOffsetEnd=357 PartOfSpeech=NNS] [Text=-LRB- CharacterOffsetBegin=358 CharacterOffsetEnd=359 PartOfSpeech=-LRB-] [Text=competently CharacterOffsetBegin=359 CharacterOffsetEnd=370 PartOfSpeech=RB] [Text=unattractive CharacterOffsetBegin=371 CharacterOffsetEnd=383 PartOfSpeech=JJ] [Text=-RRB- CharacterOffsetBegin=383 CharacterOffsetEnd=384 PartOfSpeech=-RRB-] [Text=, CharacterOffsetBegin=384 CharacterOffsetEnd=385 PartOfSpeech=,] [Text=the CharacterOffsetBegin=386 CharacterOffsetEnd=389 PartOfSpeech=DT] [Text=direction CharacterOffsetBegin=390 CharacterOffsetEnd=399 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=400 CharacterOffsetEnd=401 PartOfSpeech=-LRB-] [Text=lazy CharacterOffsetBegin=401 CharacterOffsetEnd=405 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=406 CharacterOffsetEnd=409 PartOfSpeech=CC] [Text=unimaginative CharacterOffsetBegin=410 CharacterOffsetEnd=423 PartOfSpeech=JJ] [Text=-RRB- CharacterOffsetBegin=423 CharacterOffsetEnd=424 PartOfSpeech=-RRB-] [Text=, CharacterOffsetBegin=424 CharacterOffsetEnd=425 PartOfSpeech=,] [Text=but CharacterOffsetBegin=426 CharacterOffsetEnd=429 PartOfSpeech=CC] [Text=to CharacterOffsetBegin=430 CharacterOffsetEnd=432 PartOfSpeech=TO] [Text=me CharacterOffsetBegin=433 CharacterOffsetEnd=435 PartOfSpeech=PRP] [Text=that CharacterOffsetBegin=436 CharacterOffsetEnd=440 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=441 CharacterOffsetEnd=443 PartOfSpeech=VBZ] [Text=all CharacterOffsetBegin=444 CharacterOffsetEnd=447 PartOfSpeech=DT] [Text=irrelevant CharacterOffsetBegin=448 CharacterOffsetEnd=458 PartOfSpeech=JJ] [Text=compared CharacterOffsetBegin=459 CharacterOffsetEnd=467 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=468 CharacterOffsetEnd=470 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=471 CharacterOffsetEnd=474 PartOfSpeech=DT] [Text=director CharacterOffsetBegin=475 CharacterOffsetEnd=483 PartOfSpeech=NN] [Text='s CharacterOffsetBegin=483 CharacterOffsetEnd=485 PartOfSpeech=POS] [Text=insult CharacterOffsetBegin=486 CharacterOffsetEnd=492 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=493 CharacterOffsetEnd=495 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=496 CharacterOffsetEnd=499 PartOfSpeech=DT] [Text=town CharacterOffsetBegin=500 CharacterOffsetEnd=504 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=505 CharacterOffsetEnd=508 PartOfSpeech=CC] [Text=its CharacterOffsetBegin=509 CharacterOffsetEnd=512 PartOfSpeech=PRP$] [Text=people.Thus CharacterOffsetBegin=513 CharacterOffsetEnd=524 PartOfSpeech=NNP] [Text=the CharacterOffsetBegin=525 CharacterOffsetEnd=528 PartOfSpeech=DT] [Text=invitation CharacterOffsetBegin=529 CharacterOffsetEnd=539 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=540 CharacterOffsetEnd=542 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=543 CharacterOffsetEnd=545 PartOfSpeech=PRP$] [Text=summary CharacterOffsetBegin=546 CharacterOffsetEnd=553 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=554 CharacterOffsetEnd=556 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=557 CharacterOffsetEnd=560 PartOfSpeech=DT] [Text=director CharacterOffsetBegin=561 CharacterOffsetEnd=569 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=570 CharacterOffsetEnd=573 PartOfSpeech=CC] [Text=writer CharacterOffsetBegin=574 CharacterOffsetEnd=580 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=580 CharacterOffsetEnd=581 PartOfSpeech=,] [Text=Mark CharacterOffsetBegin=582 CharacterOffsetEnd=586 PartOfSpeech=NNP] [Text=Herman CharacterOffsetBegin=587 CharacterOffsetEnd=593 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=593 CharacterOffsetEnd=594 PartOfSpeech=,] [Text=to CharacterOffsetBegin=595 CharacterOffsetEnd=597 PartOfSpeech=TO] [Text=leave CharacterOffsetBegin=598 CharacterOffsetEnd=603 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=604 CharacterOffsetEnd=607 PartOfSpeech=DT] [Text=north CharacterOffsetBegin=608 CharacterOffsetEnd=613 PartOfSpeech=NN] [Text=east CharacterOffsetBegin=614 CharacterOffsetEnd=618 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=618 CharacterOffsetEnd=619 PartOfSpeech=.] 
root(ROOT-0, stupid-7)
prep(stupid-7, In-1)
det(film-3, this-2)
pobj(In-1, film-3)
nsubj(stupid-7, they-4)
cop(stupid-7, are-5)
dep(stupid-7, all-6)
punct(stupid-7, ,-8)
conj(stupid-7, violent-9)
punct(stupid-7, ,-10)
conj(stupid-7, thieving-11)
punct(stupid-7, ,-12)
nsubj(suppose-14, thugs.I-13)
conj(stupid-7, suppose-14)
nsubj(comment-17, I-15)
aux(comment-17, could-16)
ccomp(suppose-14, comment-17)
prep(comment-17, on-18)
det(plot-20, the-19)
pobj(on-18, plot-20)
punct(predictable-22, -LRB--21)
dep(plot-20, predictable-22)
punct(predictable-22, -RRB--23)
punct(stupid-7, ,-24)
det(performances-26, the-25)
conj(stupid-7, performances-26)
punct(unattractive-29, -LRB--27)
advmod(unattractive-29, competently-28)
dep(performances-26, unattractive-29)
punct(unattractive-29, -RRB--30)
punct(stupid-7, ,-31)
det(direction-33, the-32)
conj(stupid-7, direction-33)
punct(lazy-35, -LRB--34)
appos(direction-33, lazy-35)
cc(lazy-35, and-36)
conj(lazy-35, unimaginative-37)
punct(lazy-35, -RRB--38)
punct(stupid-7, ,-39)
cc(stupid-7, but-40)
prep(irrelevant-46, to-41)
pobj(to-41, me-42)
nsubj(irrelevant-46, that-43)
cop(irrelevant-46, is-44)
advmod(irrelevant-46, all-45)
conj(stupid-7, irrelevant-46)
prep(irrelevant-46, compared-47)
pcomp(compared-47, to-48)
det(director-50, the-49)
poss(insult-52, director-50)
possessive(director-50, 's-51)
pobj(to-48, insult-52)
prep(insult-52, to-53)
det(town-55, the-54)
pobj(to-53, town-55)
cc(insult-52, and-56)
poss(people.Thus-58, its-57)
conj(insult-52, people.Thus-58)
det(invitation-60, the-59)
dep(people.Thus-58, invitation-60)
prep(invitation-60, in-61)
poss(summary-63, my-62)
pobj(in-61, summary-63)
prep(invitation-60, to-64)
det(director-66, the-65)
pobj(to-64, director-66)
cc(director-66, and-67)
conj(director-66, writer-68)
punct(invitation-60, ,-69)
nn(Herman-71, Mark-70)
appos(invitation-60, Herman-71)
punct(invitation-60, ,-72)
aux(leave-74, to-73)
infmod(invitation-60, leave-74)
det(east-77, the-75)
amod(east-77, north-76)
dobj(leave-74, east-77)
punct(stupid-7, .-78)

